File: Modul 4/Soal 4/PRAK304-2210817220007-AISYA ANINDIA SEPTHA.c
#include<stdio.h>
int main()
{
for (int b=1; b<=5; b++)
{int a;
printf("");
scanf("%d", &a);

if (a == 0)
    {printf("Nol");}

else if (a >= 1 && a <= 9)
    {printf("Satuan");}

else if (a >= 11 && a <= 19)
    {printf("Belasan");}

else if (a == 10)
    {printf("Puluhan");}

else if (a >= 20 && a <= 99)
     {printf("Puluhan");}

else {printf("Anda Menginput Melebihi Limit Bilangan");}

}
}

File: Modul 4/Soal 4/PRAK304-2210817220007-AISYA ANINDIA SEPTHA.py
for a in range (5) :
    b = int (input())
    if b > 99:
        print("Anda Menginput Melebihi Limit bilangan\n")
    elif b >= 20 :
        print("Puluhan\n")
    elif b >= 11 :
        print("Belasan\n")
    elif b == 10 :
        print("Puluhan\n")
    elif b >= 1 :
        print("Satuan\n")
    elif b == 0 :
        print("Nol\n")
